### PR TITLE
Add global hooks

### PR DESCRIPTION
**Pull request by ibolton336**
-  Make hooks 1st class objects with CRUD operations from main nav
-  Allow global hooks to be associated with a plan at plan creation time & during edit

### CODE REVIEW

File: src/app/home/pages/HooksPage/components/AddEditClusterModal/AddEditClusterForm.tsx [hunk 1]
```diff
@@ -0,0 +1,387 @@
+import React, { useState } from 'react';
```
Comment (pranavgaikwad) [inline src/app/home/pages/HooksPage/components/AddEditClusterModal/AddEditClusterForm.tsx:1]: I couldn't find any cluster actions on Hooks page. I didnt understand the usage of the Cluster Edit modals and forms in the context of hooks.

**Review by pranavgaikwad — DISMISSED** [review]
Overall, the code looks good to me. Works as expected. One small question about usage of cluster edit forms in the context of Hooks.

File: src/app/home/pages/HooksPage/components/AddEditClusterModal/AddEditClusterForm.tsx (hunk 1 above)
Comment (ibolton336) [reply src/app/home/pages/HooksPage/components/AddEditClusterModal/AddEditClusterForm.tsx]: Good catch @pranavgaikwad - removed these files.

Comment (pranavgaikwad) [reply src/app/home/pages/HooksPage/components/AddEditClusterModal/AddEditClusterForm.tsx]: Ahh I see. They were pasted from the directory.

Comment (ibolton336) [reply src/app/home/pages/HooksPage/components/AddEditClusterModal/AddEditClusterForm.tsx]: @pranavgaikwad everything seems to be working after removing those files. Yeah i forgot to remove those after planning to use them as templates.

**Review by pranavgaikwad — APPROVED** [review]
ACK

**Comment by ibolton336** [comment]
Closes #920

**Pull request merged**